src/Views/blades/pb-footer.blade.php

```diff
@@ -3,13 +3,10 @@
 		<div class="pull-left">
 			<ul class="list-inline" style="padding-top:1em;">
 				<li>
-					<a href="{{env('APP_URL')}}/feedback">Feedback</a>
-				</li>
-				<li>
 					<a href="{{env('APP_URL')}}/release">Release Information</a>
 				</li>
 				<li>
-					<a href="{{env('APP_URL')}}/help">Help</a>
+					<a href="{{env('APP_URL')}}/docs">Documentation</a>
 				</li>
```

Comment:
#4 adds some boilerplate for a "Documentation" page, we should think about how to automatically pull release notes from GH's tags for the "Release" page